modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/view_calendar_menus.jspf

```diff
@@ -563,6 +563,8 @@
 				}
 			}
 		);
+
+		addOtherCalendarInput.removeAttribute('aria-expanded');
```

Comment:
Hey @NemethNorbert, thanks for this PR!

I just have one question:

Is there any specific reason that motivated you to make this modification only in `view_calendar_menus.jspf`
 and not in yui3.

As this `aria-expanded` property is not normally used with search entries, can we go to the file that introduced this property to resolve elsewhere in the portal that are using this `autocomplete-list.js` component?

Let me know if there are any issues that I haven't noticed.

Reply:
Hey @igor-franca,

My motivation lied in how old this calendar implementation and its dependancy yui3 is. Looking at the yui3 repository, we rarely make changes there, and it seemed an easier and more cost effective approach, than updating yui3 and alloyUI to have this fix delivered for the calendar. As most of our frontend infrastructure uses clayUI, and I could not find any other usage trough the portal, while I was looking through our aui:script tags.

Thanks

Reply:
> Hey @igor-franca,
> 
> My motivation lied in how old this calendar implementation and its dependancy yui3 is. Looking at the yui3 repository, we rarely make changes there, and it seemed an easier and more cost effective approach, than updating yui3 and alloyUI to have this fix delivered for the calendar. As most of our frontend infrastructure uses clayUI, and I could not find any other usage trough the portal, while I was looking through our aui:script tags.
> 
> Thanks

Hey @NemethNorbert. Even if you haven't found places other than the Calendar that use this aui:script, the ideal would be to correct the problem directly at the source. So could you close this PR and send another one to yui3? Thanks!

Reply:
https://github.com/liferay/yui3/pull/38

Reply:
Hey @NemethNorbert, thanks for letting us know that only critical patches are accepted in that repository. I've sent this PR to BChan [here](https://github.com/brianchandotcom/liferay-portal/pull/142275).

Reply:
Thank @naraandrade ,

I close the PR at YUI3